public/app/features/trails/autoQuery/getAutoQueriesForMetric.ts

```diff
@@ -1,11 +1,13 @@
+import { isValidLegacyName } from '@grafana/prometheus/src/utf8_support';
+
 import { createDefaultMetricQueryDefs } from './queryGenerators/default';
 import { createHistogramMetricQueryDefs } from './queryGenerators/histogram';
 import { createSummaryMetricQueryDefs } from './queryGenerators/summary';
 import { AutoQueryContext, AutoQueryInfo } from './types';
 import { getUnit } from './units';
 
 export function getAutoQueriesForMetric(metric: string): AutoQueryInfo {
-  const isUtf8Metric = false;
+  const isUtf8Metric = !isValidLegacyName(metric);
```

Comment:
Given the abundance of:
```ts
const isUtf8Metric = !isValidLegacyName(metric);
```
in the various UTF-8 PRs, I wonder if a `isUtf8Metric` util would be a more helpful export than `isValidLegacyName`.

Reply:
it is definitely more helpful. But since my big PR has these names in the first place, to prevent my own confusion I changed nothing. When everything is in place, I'll rename things properly. In the end, you know, naming is hard :D